### DISCUSSION TITLE
Add Pod: Improve Extend Taxonomy experience w/ Disabled option and direction to enable component

### DISCUSSION THREAD

**Discussion by salidoraul** [pods-framework/pods#6831] [Q&A]
I was trying to extend the 'Category' taxonomy for posts and I realized that the drop-down on ''Extend an Existing Content Type' does not show Taxonomies as an option anymore. I'm on WP 3.6.1 and Pods 2.3.12.

![pods](https://f.cloud.github.com/assets/1322357/1377797/fca26170-3ab0-11e3-97ec-e37d7120fc0a.png)

**Comment by sc0ttkclark** [comment]
We need to add a disabled selection here that directs the user to Pods Admin > Components > Table Storage, for extending existing taxonomies with custom fields. You can create custom taxonomies fine because those can be registered by Pods, but since Pods doesn't currently allow for 'overriding' of post type or taxonomy options themselves, the only feature Pods has for extending taxonomies is to extend them with fields, which require the Table Storage component for creating the table necessary.
